From 92d7ec1d71e351f11ba503369eb78225510cfcc7 Mon Sep 17 00:00:00 2001
From: Hans Ulli Kroll <ulli.kroll@googlemail.com>
Date: Mon, 5 Mar 2018 18:09:10 +0100
Subject: [PATCH] leds: Fix wrong dmi_match on PC Engines APU LEDs

BIOS on APU board doesn't expose board_name property, and thus
we have to rely on the product_name instead.

Signed-off-by: Hans Ulli Kroll <ulli.kroll@googlemail.com>
Signed-off-by: Jacek Anaszewski <jacek.anaszewski@gmail.com>
---
 drivers/leds/leds-apu.c | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

--- a/drivers/leds/leds-apu.c
+++ b/drivers/leds/leds-apu.c
@@ -223,7 +223,7 @@ static int __init apu_led_probe(struct p
 
 	apu_led->pdev = pdev;
 
-	if (dmi_match(DMI_BOARD_NAME, "APU")) {
+	if (dmi_match(DMI_PRODUCT_NAME, "APU")) {
 		apu_led->profile = apu1_led_profile;
 		apu_led->platform = APU1_LED_PLATFORM;
 		apu_led->num_led_instances = ARRAY_SIZE(apu1_led_profile);
